test/end-to-end/pages/screeners/willoriginal.js

```diff
@@ -1,17 +1,22 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/screeners/willoriginal');
+const willOriginalEn = require('app/resources/en/translation/screeners/willoriginal');
+const willOriginalCy = require('app/resources/cy/translation/screeners/willoriginal');
+const contentEn = require('app/resources/en/translation/common');
+const contentCy = require('app/resources/cy/translation/common');
 
-module.exports = async function(answer) {
+module.exports = async function(language ='en', answer) {
     const I = this;
+    const commonContent = language === 'en' ? contentEn : contentCy;
+    const willOriginalContent = language === 'en' ? willOriginalEn : willOriginalCy;
 
     await I.checkPageUrl('app/steps/ui/screeners/willoriginal');
-    await I.waitForText(content.question);
+    if (language === 'en') {
+        await I.waitForText(willOriginalContent.question);
+    }
 
```

Comment:
agree with Rob think we need a welsh language check, conversation marked as resolved, but don't see how its resolved - in fact if you took out out the if statement here I would think it would work for both